README.md

```diff
@@ -10,4 +10,396 @@ TODO: description
 
 ## API Routes
 
-TODO: document API routes
+### Shared Account
+
+#### `POST /api/accounts` - Create an new shared account
+
+**Body**
+
+- `name` _{string}_ - The account owner's name
```

Comment:
Do we want to have the account owner's legal name, meaning should we have first name and last name? Or should we keep it informal?

Reply:
I think legal name would be best, personally

Reply:
This is a good question. I think it depends on how we plan to display it. If we only ever show the full name there is no benefit to breaking it up -- only makes the code more complicated and adds one more step for the user. This format also lets the user select whatever format they would prefer. If we have any place where we would like to only show part of the name then we should break it into separate fields. What do you think?

Reply:
I think we should just keep it as one field, assuming users will enter their full name?